projects/volcano/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2025 Google LLC
```

Comment:
Why not the upstream?

Reply:
We are waiting for the fuzzers to be merged. I am just using my fork to demonstrate the CI is green. This is ready once the upstream fuzzers have been merged. I will let you know.

Reply:
updated to the upstream git repo